classes/class-wc-connect-nux.php

```diff
@@ -216,34 +215,6 @@ public function register_new_carrier_dhl_pointer( $pointers ) {
 			return $pointers;
 		}
 
-		/**
-		 * Check that the current user is the owner of the Jetpack connection
-		 * - Only that person can accept the TOS
-		 *
-		 * @uses self::get_jetpack_install_status()
-		 *
-		 * @return bool
-		 */
-		public function can_accept_tos() {
```

Comment:
Since the logic now boils down to "is offline mode or is current user connected", I am inlining this logic in `set_up_nux_notices()` instead.